docker-hub/api/latest.yaml

```
@@ -521,3 +608,165 @@ paths:
           description: Forbidden - this API is only available to users on Pro or Team plans
           schema:
             $ref: '#/definitions/ErrorResponse'
+  /v2/auditlogs/{namespace}:
+    get:
+      summary: Returns list of audit log  events.
+      description: Get audit log events.
```

```suggestion
      description: Get audit log events for a given namespace.
```
The difference between "events" and "actions" (and their specific endpoints) wasn't immediately clear to me.
My understanding is that "events" refers to actual activity logged related to a given namespace. Whereas "actions" endpoint would be used as a reference point (response is general, not specific to a given namespace).

---

It took me a bit to understand this as well. The `actions` endpoint returns a list of actions that can be used in the filter for the endpoint that returns the actual events. Currently, the `namespace` portion is not used, but it is set up so that specific actions can be returned depending on the use case. If you are querying repo events, it will return the repo actions, if you want org/team member events it will return those actions. Currently, audit logs only supports repo/org events though, which use the same namespace.